src/fuzzy_match.cc

```diff
@@ -268,15 +261,15 @@ namespace fuzzy
     }
 
     /* sort the subsequences by idf weight */
-    std::priority_queue<Subseq, std::vector<Subseq>, CompareSubseq> subseq_queue;
+    std::priority_queue<Subseq, std::vector<Subseq>> subseq_queue;
```

Comment:
Can you omit the `std::vector<Subseq>` template parameter ?

Reply:
Good catch.